### DISCUSSION TITLE
about-perfectionism/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#49] [Announcements]
# about-perfectionism/

چند وقته به کمال‌گرایی فکر می‌کنم. می‌دونیم که در عین این که کمال‌گرایی افراطی چندان خوب نیست، کمال‌گرایی حداقلی مزایایی هم داره. (وگرنه چیزی که فقط مضر باشه، در چرخه تکامل به نابودی خواهد گرایید).
اما آیا در عمل هم همین‌طوره؟ آیا کمال گرایی بیشتر به سمت عمل‌گرایی می‌ره یا بی‌عملی؟
به نظرم بخشی از کمال گرایی از این میاد که ما خروجی نهایی و چکش‌کاری شده کار بقیه رو می‌بینیم و انتظار داریم که بتونیم همون اول بهترش رو انجام بدیم!

https://aprd.ir/about-perfectionism/

**Comment by hiwasham** [comment]
متن رو که میخوندم میومدم پایین منتظر بودم به ترس از شکست یا چنین مواردی برسم که اشاره کردی. 
خوشبختانه چنین موضوعاتی در حوزه روانشناسی خیلی تحلیل و بررسی شدن و مسئله به خوبی شناخته شده ست.
غالباً ریشه چنین رفتارهایی در یک ترس نهفته ست. مثلا ترس از ریجکت شدن (اگه 20 نگیرم مامانم سرد میشه باهام یا اگه مقاله من توی فلان کنفرانس چاپ بشه، آدمای متخصص این حوزه جدیم نمیگیرن) یا ترس از آسیب دیدن (اگه 20 نشم مامانم دعوام میکنه یا اگه مقاله خوبی چاپ نکنم نقدم میکنن و پشت سرم صحبت میشه)
هر کدوم از اینها هم نشون میده که ما یه زخم هایی داریم. مثلاً زخم قضاوت. یعنی انقدر به خاطر نمره 18 و 16، احساس بی ارزشی و ناکافی بودن و بد بودن بهمون دادن که الان هر نقد کوچکی هم باعث میشه اون احساسها زنده بشن. انگار یه زخم قدیمی باز شده و من 25 ساله مثل اون بچه 8 ساله به خاطر نمره 16 داره زیر سوال میره.
چاره چیه؟ از همون کودکی گفتیم بار بعدی باید 20 بشم تا این همه دردسر درست نشه و تازه کلی هم تحسین بشم.
این 20 رو تعمیم دادیم به مقاطع بعدی: رتبه کنکور و رنکینگ دانشگاه و جورنال و ...
اینها از همون کودکی و نوجوانی به صورت مکرر تجربه میشن
در واقع شرطی شدن هم در کودکی به تحلیل بالا (در مورد ترس و زخم) اضافه میشه و این رو تقویت میکنه.
به خاطر گیر افتادن توی این چرخه هایی که با این ترسها و الگوهای رفتاری ساخته میشه، مسیر طبیعی و ارگانیگ توسعه و رشد توی فضاهای پیچیده تر هم مختل میشه. یعنی دانشجو که میشیم یاد تمام تلاشمون اینه که 20 بگیریم در نتیجه روش کار توی این محیط جدید و پیچیده تر رو یاد توسعه نمیدیم.
تمام انرژی و وقتمون هم صرف این تقلای غیرعادی در این محیط جدید میشه. 
بدیهیه که کارآمد هم عمل نمیکنیم و به اون کمالی که مدنظرمون هست (که یه تصویر غیرواقع بینانه و اشتباهی هم هست از کمال) نمیرسیم.

لابلای کارها این پست رو خوندم و کلی پر حرفی کردم.
مخلصم :)

**Reply by pourmand1376** [reply]
سلام هیوا جان. ممنون که دیدگاهت رو نوشتی. چه نگاه جالبی به موضوع داری. فکر نمی‌کردم الگوهای رفتاری دوران کودکی (و نوجوانی) به این هم ربط پیدا کنند. به نظر می‌رسه که که اگر مدرسه نمی‌رفتیم و  خودمون تو خونه کتاب‌های مدرسه رو می‌خوندیم، شاید چیز زیادی از دست نمی‌دادیم (پدیده‌ای که الان خیلی‌ والدین انجامش میدن - Home Schooling). 

از چند سال پیش یه سری فیلم از سینمای هند (بالیوود) دیدم و داشتم به شباهت‌های رفتاری هندی‌ها با ما فکر می‌کردم (اکثراً فیلم‌های Aamir Khan). به نظرم سینمای هند به ما خیلی شبیه‌تر از کشوری مثل آمریکا هست. مثلاً نگاه سنتی پدرسالانه تو هند خیلی پررنگ‌تر از آمریکاست. 

این‌طور که از سینمای هند به نظر میاد، تو مدارس هند هم مثل ما تأکید خیلی زیادی روی موفقیت میشه. بچه‌ها واو به واو کتاب‌ها رو حفظ می‌کنند و تمام عمرشون تلاش می‌کنند که به خواسته‌های پدر و مادرشون برسند. 

اونجا هم میشه نمونه‌های زیادی از رفتار کمال‌گرایانه رو دید. انقدر آدم‌ها دنبال اول شدن و بهترین بودن هستند که فراموش می‌کنند که تنها هدف زندگی، اول شدن نیست. یعنی به نظر میاد بخشی از این قضیه کلاً به ترتیب دوران کودکی و «فرهنگ رایج» بستگی داره. بعید می‌دونم این تب و تاب موفقیت در ما، تو یه کشور اروپایی هم به همین شکل باشه.

**Reply by hiwasham** [reply]
در مورد مسئله مدرسه، فکر کنم این برات جالب باشه:
https://supermemo.guru/wiki/Unschooling
https://supermemo.guru/wiki/Problem_of_Schooling
البته این یه سرنخه به این سایت و Piotr Woźniak
فیلم مشق شب کیارستمی هم احتمالا برات جالب باشه

در مورد شباهت سیستم ایرات و هند هم موافقم. ترکیبه و یه سری کشورهای اطراف خودمون هم شباهت های زیادی به ما دارن.
